chicago-notes.csl

```diff
@@ -0,0 +1,4726 @@
+        </group>
+      </else-if>
+      <else>
+        <group delimiter=", ">
+          <text macro="author-note"/>
+          <text macro="title-and-source-note"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-notes-full-subsequent-author-title">
+    <choose>
+      <if position="subsequent">
+        <text macro="citation-notes-shortened-author-title"/>
+      </if>
+      <else>
+        <text macro="citation-notes-full"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-notes-shortened-author-title">
+    <!-- Shortened note styles mirror the filtering approach of author-date -->
+    <choose>
+      <if match="any" variable="part-number volume">
+        <!-- comma always needed with volume/part -->
+        <group delimiter=", ">
+          <text macro="citation-notes-shortened-author-title-item"/>
+          <text macro="source-locator-subsequent"/>
+        </group>
+      </if>
+      <else-if locator="section" type="classic">
+        <!-- `section` is the closest CSL locator to a classical reference -->
+        <group delimiter=" ">
+          <text macro="citation-notes-shortened-author-title-item"/>
+          <text macro="source-locator-subsequent"/>
+        </group>
+      </else-if>
+      <else>
+        <group delimiter=", ">
+          <text macro="citation-notes-shortened-author-title-item"/>
+          <text macro="source-locator-subsequent"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-notes-shortened-author-title-item">
+    <group delimiter=" ">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <choose>
+          <if match="none" type="bill hearing legal_case legislation regulation treaty">
+            <text macro="title-and-descriptions-short"/>
+          </if>
+        </choose>
+        <text macro="citation-notes-shortened-author-title-disambiguate"/>
+        <choose>
+          <if match="any" type="interview personal_communication">
+            <choose>
+              <if match="none" variable="archive archive-place container-title DOI number publisher references URL">
+                <!-- unpublished `interview` or `personal_communication` use inline format (CMOS 14.111) -->
+                <text macro="date-short"/>
+              </if>
+            </choose>
+          </if>
+        </choose>
+        <text macro="source-monographic-publication-short-unbracketed"/>
+      </group>
+      <text macro="source-monographic-publication-short-bracketed"/>
+    </group>
+  </macro>
+  <macro name="citation-notes-shortened-author-title-disambiguate">
+    <!-- CMOS gives examples of added dates (CMOS 14.13) and editors/translators, publisher, date (CMOS 14.146) -->
+    <group delimiter=", ">
+      <choose>
+        <if disambiguate="true">
+          <choose>
+            <if disambiguate="true">
+              <text macro="label-version"/>
```

Comment:
This second test for `disambiguate` is unnecessary

Reply:
I'm not following. The conditions for the two tests are identical `<if disambiguate="true">`